app/controllers/lease_ups_controller.rb

```diff
@@ -0,0 +1,21 @@
+class LeaseUpsController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    full_listing = listing_service.listing(params[:listing_id], true) # true?
```

Comment:
Re: true, if you look at request.rb, `force` refers to not using caching. Let's remove this for now. We can always switch by updating the `CACHE_SALESFORCE_REQUESTS` env var.